### ISSUE TITLE
False positive unused return value warning for Enum.sum and Tuple functions

### ISSUE THREAD

**Issue by amitkt2020** [rrrene/credo#546]
### Environment

* Credo version (`mix credo -v`): 0.9.2
* Erlang/Elixir version (`elixir -v`):
Erlang/OTP 20 [erts-9.3] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10] [hipe] [kernel-poll:false]
Elixir 1.6.4 (compiled with OTP 20)
* Operating system: CentOS 7

### What were you trying to do?
Getting incorrect results when using Enum.sum and Tuple.duplicate with :ets.insert

Examples:
```
   :ets.insert(table, {:total_sum, Enum.sum([1,2,3])})

   :ets.insert(table, {:some_values, Tuple.duplicate(1, 10)})
```

### Expected outcome
No warning.

### Actual outcome
[W] ↗ There should be no unused return values for Enum.sum().
[W] ↗ There should be no unused return values for Tuple functions.

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by amitkt2020** [comment]
Thank you, Issue is resolved 👍
